lib/helper/Polly.js

```diff
@@ -73,14 +73,12 @@ class Polly extends Helper {
     await page.setRequestInterception(true);
 
     this.polly = new PollyJS(title, {
+      mode: 'passthrough',
```

Comment:
This will enable by default passthrough all request.
Refer to [this](https://github.com/Netflix/pollyjs/issues/224#issuecomment-509041054).